### ISSUE TITLE
cnpm中自动添加‘\9‘导致Chrome中报错

### ISSUE THREAD

**Issue by sundjly** [mzohaibqc/antd-theme-generator#15]
cnpm中自动添加‘\9‘导致Chrome中报错
![image](https://user-images.githubusercontent.com/23523003/50834204-60b0e400-138e-11e9-9be7-2ccd479cdfe7.png)

**Comment by sundjly** [comment]
目前通过锁定版本解决了
```
 "antd": "3.10.7",
 "antd-theme-generator": "1.1.3",
```

**Comment by mzohaibqc** [comment]
@sundjly This is due to Ant Design popover style error
ant-design/ant-design#14148
https://github.com/ant-design/ant-design/blob/master/components/popover/style/index.less#L58
```
  &-inner {
    background-color: @popover-bg;
    background-clip: padding-box;
    border-radius: @border-radius-base;
    box-shadow: @box-shadow-base;
    box-shadow: ~'0 0 8px @{shadow-color} \9';
  }
```

You can use some previous version of `antd` to fix this issue if you are not using any features or fixes in this release

**Comment by mzohaibqc** [comment]
@chenshuai2144 Just FYI

**Comment by afc163** [comment]
It is a IE9 css hacks, we can't remove it if there is another solution.

**Comment by mzohaibqc** [comment]
@afc163 Thanks for letting me know. I'll handle this somehow.

**Comment by mzohaibqc** [comment]
@sundjly Above mentioned issue has been fixed in `antd-theme-generator@1.1.6` so update your project dependency to fix this issue.

**Comment by sundjly** [comment]
Thanks for letting me know.
